apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -1053,6 +1061,79 @@ static int _wm_test_join(struct options *opt, int argc, char *argv[])
 	return 0;
 }
 
+static int _wm_test_stress(struct options *opt, int argc, char *argv[])
+{
+	if (argc < 4) {
+		return -1;
+	}
+
+	opt->stress_tc_idx = atoi(argv[3]);
+	if (opt->stress_tc_idx == 1) {
+		// TC index is 1
+		if (argc != 7 && argc != 6) {
+			return -1;
+		}
+	} else if (opt->stress_tc_idx == 2){
+		if (argc != 10 && argc != 9) {
+			return -1;
+		}
+	} else {
+		return -2;
+	}
+
+	opt->ssid = argv[4];
+	opt->auth_type = get_auth_type(argv[5]);
+	if (opt->auth_type == WIFI_MANAGER_AUTH_OPEN || opt->auth_type == WIFI_MANAGER_AUTH_IBSS_OPEN) {
+		// case: open mode
+		opt->password = "";
+		opt->crypto_type = WIFI_MANAGER_CRYPTO_NONE;
+		return 0;
+	}
+
+	if (argc == 6) {
+		// case: unspecified security mode
+		opt->auth_type = WIFI_MANAGER_AUTH_UNKNOWN;
+		opt->crypto_type = WIFI_MANAGER_CRYPTO_UNKNOWN;
+		opt->password = argv[5];
+		return 0;
+	}
+
+	// case: security mode + password
+	if (opt->auth_type == WIFI_MANAGER_AUTH_UNKNOWN) {
+		return -1;
+	}
+
+	if (opt->auth_type == WIFI_MANAGER_AUTH_WEP_SHARED) {
+		if (strlen(argv[6]) == 13) {
+			opt->crypto_type = WIFI_MANAGER_CRYPTO_WEP_128;
+		} else if (strlen(argv[6]) == 5) {
+			opt->crypto_type = WIFI_MANAGER_CRYPTO_WEP_64;
+		} else {
+			return -1;
+		}
+	} else {
+		opt->crypto_type = get_crypto_type(argv[5]);
+		if (opt->crypto_type == WIFI_MANAGER_CRYPTO_UNKNOWN) {
+			return -1;
+		}
+	}
+	opt->password = argv[6];
+
+	int softap_index = 7;
+	if (opt->auth_type == WIFI_MANAGER_AUTH_OPEN) {
+		softap_index = 6;
+	}
```

Comment:
If auth_type is OPEN, the `softap_index` will be set twice, 7 and then 6.
How about below?
```
if (auth_type == OPEN) {
   softap_index = 6;
} else {
   softap_index = 7;
}

Reply:
Anyway softap_index should be initialized before it is allocated. so I think this way is ok.